driver/rockcraft.yaml

```
@@ -0,0 +1,26 @@
+name: knative-serving-deriver
+base: ubuntu@24.04
+version: v2.2.0
+summary: Kubeflow Pipelines Driver
+description: This image is used as part of the Charmed Kubeflow product
+license: Apache-2.0
+
+platforms:
+  amd64:
+
+services:
+  driver:
+    override: merge
+    command: "driver"
```

```suggestion
    command: "/bin/driver"
```
better not to diverge from the upstream [entrypoint](https://github.com/kubeflow/pipelines/blob/2.2.0/backend/Dockerfile.driver#L42)